src/Edm/Validation/Internal/InterfaceValidator/VisitorOfIPropertyConstructor.php

```diff
@@ -10,12 +10,12 @@
 
 class VisitorOfIPropertyConstructor extends VisitorOfT
 {
-    protected function VisitT($expression, array &$followup, array &$references): iterable
+    protected function VisitT($expression, array &$followup, array &$references): ?iterable
     {
         assert($expression instanceof IPropertyConstructor);
-        $errors = null;
+        $errors = [];
 
-        if ($expression->getName() == null) {
+        if (null === $expression->getName()) {
```

Comment:
And yet more nullability confusion, @c-harris .  Nullable here, not nullable in EdmModelCsdlSchemaWriter->WritePropertyConstructorElementHeader and ExpressionTypeChecker::TryAssertRecordAsType, nullability ambiguous in RecordExpressionHelpers->FindProperty,